bucket/dreamset.json

```diff
@@ -0,0 +1,37 @@
+{
+    "version": "2.4.13",
+    "description": "Dreamset is a settings editor for digital satellite receivers.",
+    "homepage": "http://www.dreamset-editor.com",
+    "license": "Unknown",
+    "architecture": {
+        "64bit": {
+            "url": "http://www.dreamset-editor.com/files/dreamset2413x64.zip",
+            "hash": "sha1:b24d4a3d0b1802b90ca6423d960c5b8cc367f6e2"
+        },
+        "32bit": {
+            "url": "http://www.dreamset-editor.com/files/dreamset2413.zip",
+            "hash": "sha1:522733f6a1103ef6dd412f981573d4706bcfa83f"
+        }
+    },
+    "bin": "dreamset.exe",
```

Comment:
```suggestion
```

Reply:
Why to remove this line? The documentation says:
"bin: A string or array of strings of programs (executables or scripts) to make available on the user's path."